src/Session/Handler/FileHandler.php

```diff
@@ -122,6 +152,7 @@ public function gc($maxlifetime)
             try {
                 $this->fs->remove($file);
             } catch (IOException $e) {
+                $this->logger->error('Unable to remove session file ' . $file);
                 return false;
```

Comment:
Doesn't matter. SessionStorage doesn't check the returns. Idk what I would do with a false return.